sites/dotnet-guide/contents/authors/pwe/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Paul Everitt
+subtitle: PyCharm Developer Advocate at JetBrains
+thumbnail: ./pwe.jpg
+---
+
+Paul is the PyCharm Developer Advocate at JetBrains. Before that, Paul
+was a co-founder of Zope Corporation, taking the first open source
+application server through $14M of funding. Paul has bootstrapped both
+the Python Software Foundation and the Plone Foundation. Prior to that,
```

Comment:
```suggestion
the [Python Software Foundation](https://www.python.org/psf/) and the [Plone Foundation](https://plone.org/foundation). Prior to that,
```

Reply:
Up to @pauleveritt , as this may need other Guides to update as well. Paul, "commit suggestion" at will.